portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,79 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
-}
+/* IDs */
 
 #greeting-container {
   margin-top: 20px;
 }
+
+#avatar {
+    width: 40%;
+}
+
+#title > * {
```

Comment:
Using the universal selector here isn't totally necessary since anchor tags (<a>) are inline by default. The universal selector (*) is bug prone and is very inefficient to evaluate.

https://www.w3schools.com/css/css_display_visibility.asp